map_width: 45
map_height: 17
x_coords:
  [
    1.3,
    2.7,
    4.0,
    5.9,
    7.8,
    9.7,
    11.6,
    13.5,
    15.4,
    17.3,
    19.2,
    21.1,
    23.0,
    24.9,
    26.8,
    28.7,
    30.6,
    32.5,
    34.4,
    36.3,
    38.2,
    40.1,
    42.0,
    43.9,
    45.8,
    47.7,
    49.6,
    51.5,
    53.4,
    55.3,
    57.2,
    59.1,
    61.0,
    62.9,
    64.8,
    66.7,
    68.6,
    70.5,
    72.4,
    74.3,
    76.2,
    78.1,
    80.0,
    81.9,
    84.2,
  ]
y_coords:
  [
    1.5,
    3.6,
    5.7,
    8.0,
    9.9,
    11.8,
    13.4,
    15.9,
    17.5,
    19.6,
    21.7,
    23.8,
    25.9,
    28.0,
    30.1,
    32.2,
    34.3,
  ]
free_grids:
  [
    2,
    3,
    4,
    5,
    6,
    7,
    8,
    9,
    10,
    11,
    12,
    13,
    14,
    15,
    16,
    17,
    18,
    19,
    20,
    47,
    49,
    51,
    53,
    55,
    57,
    59,
    61,
    63,
    65,
    92,
    94,
    96,
    98,
    100,
    102,
    104,
    106,
    108,
    110,
    137,
    139,
    141,
    143,
    145,
    147,
    149,
    151,
    153,
    155,
    156,
    157,
    158,
    159,
    160,
    161,
    162,
    163,
    164,
    165,
    166,
    167,
    168,
    169,
    170,
    171,
    172,
    180,
    181,
    182,
    184,
    186,
    188,
    190,
    192,
    194,
    196,
    198,
    200,
    201,
    202,
    203,
    204,
    205,
    206,
    207,
    208,
    209,
    210,
    211,
    212,
    213,
    214,
    215,
    216,
    217,
    225,
    226,
    227,
    228,
    229,
    230,
    231,
    232,
    233,
    234,
    235,
    236,
    237,
    238,
    239,
    240,
    241,
    242,
    243,
    244,
    245,
    246,
    247,
    248,
    249,
    250,
    251,
    252,
    253,
    254,
    255,
    256,
    257,
    258,
    259,
    260,
    261,
    262,
    270,
    271,
    272,
    273,
    274,
    275,
    276,
    277,
    278,
    279,
    280,
    281,
    282,
    283,
    284,
    285,
    286,
    287,
    288,
    289,
    290,
    291,
    292,
    293,
    294,
    295,
    296,
    297,
    298,
    299,
    300,
    301,
    302,
    303,
    304,
    305,
    306,
    307,
    308,
    309,
    310,
    315,
    317,
    319,
    321,
    323,
    325,
    327,
    329,
    331,
    333,
    335,
    337,
    339,
    341,
    343,
    345,
    347,
    349,
    351,
    353,
    354,
    355,
    356,
    357,
    358,
    359,
    360,
    362,
    364,
    366,
    368,
    370,
    372,
    374,
    376,
    378,
    380,
    382,
    384,
    386,
    388,
    390,
    392,
    394,
    396,
    398,
    400,
    402,
    404,
    405,
    407,
    409,
    411,
    413,
    415,
    417,
    419,
    421,
    423,
    425,
    427,
    429,
    431,
    433,
    435,
    437,
    439,
    441,
    443,
    445,
    447,
    449,
    450,
    452,
    454,
    456,
    458,
    460,
    462,
    464,
    466,
    468,
    470,
    472,
    474,
    476,
    478,
    480,
    482,
    484,
    486,
    488,
    490,
    492,
    494,
    495,
    497,
    499,
    501,
    503,
    505,
    507,
    509,
    511,
    513,
    515,
    517,
    519,
    521,
    523,
    525,
    527,
    529,
    531,
    533,
    535,
    537,
    539,
    540,
    542,
    544,
    546,
    548,
    550,
    552,
    554,
    556,
    558,
    560,
    562,
    564,
    566,
    568,
    570,
    572,
    574,
    576,
    578,
    580,
    582,
    584,
    585,
    587,
    589,
    591,
    593,
    595,
    597,
    599,
    601,
    603,
    605,
    607,
    609,
    611,
    613,
    615,
    617,
    619,
    621,
    623,
    625,
    627,
    629,
    630,
    632,
    634,
    636,
    638,
    640,
    642,
    644,
    646,
    648,
    650,
    652,
    654,
    656,
    658,
    660,
    662,
    664,
    666,
    668,
    670,
    672,
    674,
    675,
    677,
    679,
    681,
    683,
    685,
    687,
    689,
    691,
    693,
    695,
    697,
    699,
    701,
    703,
    705,
    707,
    709,
    711,
    713,
    715,
    717,
    719,
    720,
    721,
    722,
    723,
    724,
    725,
    726,
    727,
    728,
    729,
    730,
    731,
    732,
    733,
    734,
    735,
    736,
    737,
    738,
    739,
    740,
    741,
    742,
    743,
    744,
    745,
    746,
    747,
    748,
    749,
    750,
    751,
    752,
    753,
    754,
    755,
    756,
    757,
    758,
    759,
    760,
    761,
    762,
    763,
    764,
  ]
modified_grids:
  [
    [2, 5.0, 1.5],
    [47, 5.0, 3.6],
    [92, 5.0, 5.7],
    [137, 5.0, 8.0],
    [191, 5.0, 9.9],
    [20, 37.4, 1.5],
    [65, 37.4, 3.6],
    [110, 37.4, 5.7],
    [155, 37.4, 8.0],
    [200, 37.4, 9.9],
  ]
